src/deepforest/main.py

```diff
@@ -434,226 +422,158 @@ def predict_image(self,
         results = predict._dataloader_wrapper_(model=self,
+                                               crop_model=crop_model,
                                                trainer=self.trainer,
-                                               annotations=df,
                                                dataloader=dataloader,
-                                               root_dir=root_dir,
-                                               nms_thresh=self.config.nms_thresh,
-                                               color=color,
-                                               savedir=savedir,
-                                               thickness=thickness)
+                                               root_dir=root_dir)
 
         results.root_dir = root_dir
 
         return results
 
     def predict_tile(self,
-                     raster_path=None,
                      path=None,
+                     paths=None,
                      image=None,
                      patch_size=400,
                      patch_overlap=0.05,
                      iou_threshold=0.15,
-                     in_memory=True,
-                     return_plot=False,
-                     mosaic=True,
-                     sigma=0.5,
-                     thresh=0.001,
-                     color=None,
-                     thickness=1,
-                     crop_model=None,
-                     crop_transform=None,
-                     crop_augment=False):
+                     dataloader_strategy="single",
+                     crop_model=None):
         """For images too large to input into the model, predict_tile cuts the
         image into overlapping windows, predicts trees on each window and
         reassambles into a single array.
 
         Args:
-            raster_path: [Deprecated] Use 'path' instead
             path: Path to image on disk
+            paths: List of paths to images on disk
             image (array): Numpy image array in BGR channel order following openCV convention
             patch_size: patch size for each window
             patch_overlap: patch overlap among windows
             iou_threshold: Minimum iou overlap among predictions between windows to be suppressed
-            in_memory: If true, the entire dataset is loaded into memory
-            mosaic: Return a single prediction dataframe (True) or a tuple of image crops and predictions (False)
-            sigma: variance of Gaussian function used in Gaussian Soft NMS
-            thresh: the score thresh used to filter bboxes after soft-nms performed
+            dataloader_strategy: "single", "batch", or "window".
+                - "Single" loads the entire image into memory and passes individual windows to GPU and cannot be parallelized.
+                - "batch" loads the entire image into GPU memory and creates views of an image as batch, requires in the entire tile to fit into GPU memory. CPU parallelization is possible for loading images.
```

Comment:
There's something off about this sentence (at least as it shows in the diff). Probably need to get rid of `of` and `in`

Reply:
I think I fixed this, not 100% what this was directed to.